client/src/index.tsx

```diff
@@ -1,11 +1,17 @@
 import 'react-app-polyfill/ie11'
 import React from 'react'
 import ReactDOM from 'react-dom'
+import { BrowserRouter as Router } from 'react-router-dom'
 import './index.css'
 import App from './App'
 import * as serviceWorker from './serviceWorker'
 
-ReactDOM.render(<App />, document.getElementById('root'))
+ReactDOM.render(
+  <Router>
+    <App />
+  </Router>,
```

Comment:
why this change?

Reply:
@jonahkagan Because in order to test the route implementation I needed to change the routes, which is only possible with passing in different routes to the Router component. I couldn't do that if the Router component was included in the component I was testing.

Reply:
Makes sense. It actually mirrors the example in the docs! https://testing-library.com/docs/example-react-router

Reply:
Yup!